BepInEx.EnableResize/EnableResize.cs

```diff
@@ -62,24 +76,43 @@ internal void Awake()
             }, IntPtr.Zero);
 
             if (WindowHandle == IntPtr.Zero) return;
-            SceneManager.sceneLoaded += (s, lsm) =>
-            {
-                if (lsm == LoadSceneMode.Single) ResizeWindow();
-            };
+
+            StartCoroutine(TestScreen());
+            ConfigEnableResize.SettingChanged += (sender, args) =>  StartCoroutine(TestScreen());
```

Comment:
This line should be before line 63 or if the setting is disabled at startup this won't get called.

Reply:
That's the whole idea, to not mess up with user.dll things if disabled, probably this is why Anon emphasized that the plugin will only work after restart.

Reply:
You can initialize everything on first use instead in Awake.

Reply:
That's a good idea! Will try that!